src/routes/rules/components/RulePreview.vue

```diff
@@ -49,10 +49,6 @@ defineProps({
       type: String,
       required: true,
     },
-    icon: {
-      type: String,
-      default: '',
-    },
```

Comment:
This prop was unused so I removed it and deleted all the data that was passed to it